lib/index.js

```diff
@@ -13,6 +13,11 @@ module.exports = (logger) => {
     return (err, req, res, next) => {
         const error = err.message || err.data || err;
 
+        const {
+            status = 500,
+            code = 'ERROR_OCCURRED',
```

Comment:
Should we say INTERNAL_SERVER_ERROR instead?

Reply:
Right. Updating now. Thanks!